packages/data-point/README.md

```diff
@@ -23,14 +23,13 @@ npm install --save data-point
 - [Transforms](#transforms)
   - [dataPoint.resolve](#api-data-point-resolve)
   - [dataPoint.transform](#api-data-point-transform)
+- [Accumulator Object](#accumulator-object)
 - [Reducers](#reducers)
-  - [Accumulator](#accumulator)
   - [PathReducer](#path-reducer)
   - [FunctionReducer](#function-reducer)
   - [ObjectReducer](#object-reducer)
   - [EntityReducer](#entity-reducer)
   - [ListReducer](#list-reducer)
-  - [Collection Mapping](#reducer-collection-mapping)
```

Comment:
The `Accumulator` and `Collection Mapping` sections are not describing reducer types, so I'm moving their location in the readme